### ISSUE TITLE
Cannot load otel-webserver-module with Nginx 1.21.6

### ISSUE THREAD

**Issue by phillipuniverse** [open-telemetry/opentelemetry-cpp-contrib#201]
**Describe your environment**

Docker with `FROM nginx:1.21.6`

**Steps to reproduce**

Included the build `ngx_http_opentelemery_module.so` like this:

```
load_module modules/ngx_http_opentelemetry_module.so;
```

output was:

```
gateway_1  | 2022/07/30 22:40:55 [emerg] 28#28: module "/etc/nginx/modules/ngx_http_opentelemetry_module.so" version 1018000 instead of 1021006 in /etc/nginx/common-outer-config.nginx.conf:5
gateway_1  | nginx: [emerg] module "/etc/nginx/modules/ngx_http_opentelemetry_module.so" version 1018000 instead of 1021006 in /etc/nginx/common-outer-config.nginx.conf:5
```

Nginx then immediately died.

**What is the expected behavior?**

Nginx started.

**What is the actual behavior?**

Nginx fails to start.

**Additional context**

**Comment by phillipuniverse** [comment]
The error message (and in the [readme](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/otel-webserver-module#nginx-webserver-module)) implies that this only works with Nginx 1.18.0. According to [this endoflife.date](https://endoflife.date/nginx), 1.18 was announced end of life April 2021, over a year ago.

**Comment by DebajitDas** [comment]
Released a new version supporting stable (1.22.0) and mainline (1.23.0) https://github.com/open-telemetry/opentelemetry-cpp-contrib/releases/tag/webserver%2Fv1.0.2 of nginx. 1.18.0 is no longer supported.

**Comment by DebajitDas** [comment]
Closing this issue.
